kwok/charts/Chart.yaml

```diff
@@ -2,8 +2,8 @@ apiVersion: v2
 name: karpenter
 description: A Helm chart for Karpenter's KwoK Provider, integrated with https://github.com/kubernetes-sigs/kwok.
 type: application
-version: 0.35.0
-appVersion: 0.35.0
+version: 0.35.4
+appVersion: 0.35.4
```

Comment:
This doesn't track the current release so was unclear whether I should bump this but I've added a new version.